src/OSPSuite.Presentation/Presenters/Journal/JournalPageEditorPresenter.cs

```diff
@@ -194,7 +209,27 @@ public bool AlreadyEditing(JournalPage journalPage)
 
       public void UpdateUserPreferences()
       {
-         _userSettings.JournalPageEditorSettings.ShowTableGridLines = _view.GetGridLinesPreference();
+         _journalPageSettings.ShowTableGridLines = _view.GetGridLinesPreference();
+      }
+
+      public void NavigateToPreviousPage() => navigateToPage(-1);
+
+      public void NavigateToNextPage() => navigateToPage(1);
+
+      private void navigateToPage(int pageOffset)
+      {
+         var journalPage = _journalPageDTO?.JournalPage;
+         if (journalPage == null)
+            return;
+
+         var journal = _journalRetriever.Current;
```

Comment:
That's the real code here. Get the index of current page. Get page by index +- 1 edit page